applications/fishing-map/src/features/workspace/Workspace.tsx

```diff
@@ -117,7 +118,9 @@ function Workspace() {
   const workspace = useSelector(selectWorkspace)
   const workspaceStatus = useSelector(selectWorkspaceStatus)
   const locationCategory = useSelector(selectLocationCategory)
-  const resourceQueries = useSelector(selectDataviewsResourceQueries)
+  const resourceQueries = resolveResourcesFromDatasetConfigs(
+    useSelector(selectDataviewsForResourceQuerying) ?? []
+  )
 
   useEffect(() => {
```

Comment:
As we are refactoring this... what about having a validation to avoid requesting the same request already done with an error every time another resource changes? 
What happens now is:
1. You select a vessel and the track fails
2. You select another vessel
3. The first vessel tries to reload the track again